Libraries/LibAudio/WavLoader.cpp

```diff
@@ -33,23 +33,29 @@
 
 namespace Audio {
 
-WavLoader::WavLoader(const StringView& path)
+WavLoaderPlugin::WavLoaderPlugin(const StringView& path)
     : m_file(Core::File::construct(path))
 {
     if (!m_file->open(Core::IODevice::ReadOnly)) {
         m_error_string = String::formatted("Can't open file: {}", m_file->error_string());
         return;
     }
 
-    if (!parse_header())
+    valid = parse_header();
+    if (!valid)
         return;
 
     m_resampler = make<ResampleHelper>(m_sample_rate, 44100);
 }
 
-RefPtr<Buffer> WavLoader::get_more_samples(size_t max_bytes_to_read_from_input)
+bool WavLoaderPlugin::sniff()
 {
-#ifdef AWAVLOADER_DEBUG
+    return valid;
+}
+
+RefPtr<Buffer> WavLoaderPlugin::get_more_samples(size_t max_bytes_to_read_from_input)
+{
+#ifdef AWavLoaderPlugin_DEBUG
```

Comment:
Please don't use lowercase letters in macros. Also, if you change a debug macro, you need to update CMakeLists.txt (see the CI failure.)

Reply:
Fixed it, sorry for that.